### PR TITLE
Add user image repo refs for a few communities

### PR DESCRIPTION
**Pull request by choldgraf**
This adds user image links for a few images that we have recently had to interact with. Just a short-term improvement until we can improve this in a more sustainable fashion in the future.

### CODE REVIEW

File: config/clusters/uwhackweeks/common.values.yaml [hunk 1]
```diff
@@ -53,6 +53,7 @@ basehub:
             - name: home
               mountPath: /home/jovyan/shared
               subPath: _shared
+      # User image repo: https://github.com/ICESAT-2HackWeek/jupyter-image-2020
```
Comment (choldgraf) [inline config/clusters/uwhackweeks/common.values.yaml:56]: @scottyhq is this correct? I couldn't find any other user image for the hackweek though this one seems to be updated quite a while ago

Comment (scottyhq) [reply config/clusters/uwhackweeks/common.values.yaml:56]: we actually have a monorepo going this time around that includes content and image in one place https://github.com/ICESAT-2HackWeek/website2022

Comment (choldgraf) [inline config/clusters/uwhackweeks/common.values.yaml:56]:
```suggestion
      # User image repo: https://github.com/ICESAT-2HackWeek/website2022
```

**Review by sgibson91 — APPROVED** [review]

**Review by GeorgianaElena — APPROVED** [review]
Good idea 🚀

**Pull request merged**